ee-feature-pack/common/src/main/resources/modules/system/layers/base/io/opentelemetry/api/main/module.xml

```diff
@@ -25,6 +25,12 @@
     </properties>
     <resources>
         <artifact name="${io.opentelemetry:opentelemetry-api}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-api-logs}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-sdk}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-sdk-logs}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-sdk-metrics}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-sdk-trace}"/>
+        <artifact name="${io.opentelemetry:opentelemetry-sdk-common}"/>
```

Comment:
Is there a reason these were mostly moved out of `io.opentelemtry.trace`? This is explicitly named with "api" in the module name and these look like implementation details. Am I wrong?

Reply:
I moved those to make them visible to.. something -- I can't recall off hand; I think it's in the Micrometer PR. To be honest, these should probably be in their own module, though I'm trying to reduce the churn in this PR. If you think it's important enough, I can reorg those modules.

Reply:
It feels a little weird to put the i the API IMO. I would opt more for a `io.opentelemetry.sdk` module over moving them here. Other opinions are welcome though :)